myCTD-assignment/css/style.css

```diff
@@ -44,69 +99,81 @@ div.aboutMe{
 
 }
 
-div.exp{
+div#exp{
+    
+    display: flex;
     font-weight: bolder;
-    margin:5vh 45vw 1vh 10vw;
+    margin: 5rem 4rem 6rem 8rem;
     border:thick double;
     border-color:blue;
-    overflow:auto;
+    overflow: auto;
+    
+}
+div#exp li{
+    display: flex;
 }
 /* div.johnny5{
     margin:0rem 0rem 0rem 0rem;
     
 }  */
                         /*fix image sizing */
-div.johnny5,img{
+div#johnny5,img{
     border: 3px solid green;
     border-radius: 100px 5%;
-    padding: 5px;
+    padding: 0.1rem;
     width: 20rem;
     display:inline-block;
-    margin:1vh 5vw 0vh 5vw;
+    margin:1vh 5vw 0vh 2vw;
+    background-color:chocolate;
+    color:aliceblue;
     
 }
 
 /*To do */
-div.skillsList{
+div#skillsList{
     text-align: left;
     font-style: italic;
-    font-size:18px;
-    border:10px dashed red;
+    font-size:1rem;
+    border: 0.2rem dashed red;
     margin:3vh 55vw 2vh 10vw;
-   
+    column-count: 3;
+    display:flex;
 }
-div.connect{
+div#connect{
     margin:3vh 0vw 2vh 15vw;
 }
-div.leaveMessage{
+div#leaveMessage{
     font-family:'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
     font-size:smaller;
     font-weight:bold;
-    margin:5vh 0vw 0vh 15vw;
-    padding: auto;
+    margin:3rem 0rem 0rem 15rem;
+    padding: 2rem;
+    display: flexbox;
```

Comment:
Similar to above, adding the display: flex and justify-content: space-between does not alter the display of your form.